src/utils.js

```diff
@@ -187,10 +187,10 @@ var Utils = exports.Utils = {
    * @returns {Boolean} true if disabled
    **/
   isPrivacyBadgerEnabled: function(origin){
-    if(localStorage.disabledSites && JSON.parse(localStorage.disabledSites).length > 0){
+    if(localStorage.disabledSites){
       var sites = JSON.parse(localStorage.disabledSites);
-      for(var i = 0; i < sites.length; i++){
-        if(sites[i] === origin){ return false; }
+      if(sites[origin]){
+        return false;
```

Comment:
We can get more speed by doing so. But according to that we should write onto localStorage only before the chrome shuts down. As of now, I have no idea how to do this. If there is any, could you suggest me?